### PR TITLE
Changelog for 8.2.1.

### PR DESCRIPTION
**Pull request by pmaytak**

### CODE REVIEW

**Review by jennyf19 — APPROVED** [review]
:shipit:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,26 @@
...
+=====
+### New features
+- Update to use .NET 9 GA. See [2990](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2990).
+
+### Bug fixes
+- Remove dependency on Microsoft.Bcl.TimeProvider for .NET 8+ targets. See [2935](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2935).
+- Update cgmanifest to align with the JSON schema. See [2969](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2969).
+
+### Fundamentals
+- Streamline token creation in `SecurityTokenDescriptor`. See [2993](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/issues/2993).
```
Comment (msbw2) [inline CHANGELOG.md]:
```suggestion
- Streamline token creation by using `SecurityTokenDescriptor`. See [2993](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/issues/2993).
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,26 @@
...
+- Prevent inlining to guarantee stack frames in test. See [2999](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/issues/2999).
+
+### Work related to redesign of IdentityModel's token validation logic [#2711](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/issues/2711)
+- Simplify stack frame caching. See [2976](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2976).
```
Comment (msbw2) [inline CHANGELOG.md:17]: The name on this one was confusing, but this is more in line with what the change actually does:
```suggestion
- Add stack frame caching. See [2976](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2976).
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,26 @@
...
+- Implement reading SAML and SAML2 tokens. See [2980](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2980).
```
Comment (msbw2) [inline CHANGELOG.md]:
```suggestion
- Implement new model for reading SAML and SAML2 tokens. See [2980](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2980).
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,26 @@
...
+- Implement validating SAML signature. See [2950](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/issues/2950).
```
Comment (msbw2) [inline CHANGELOG.md]:
```suggestion
- Implement new model for validating SAML signature. See [2950](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/issues/2950).
```

File: CHANGELOG.md (hunk 2 above)
Comment (iNinja) [reply CHANGELOG.md:17]: We had stack frame caching before. This PR simplified the way we did it.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,26 @@
...
+- Add tests for `IssuerExtensibility`. See [2987](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2987).
+- Add validation for SAML and SAML2 issuer signing key. See [2965](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2965).
```
Comment (msbw2) [inline CHANGELOG.md]:
```suggestion
- Switch to new validation model for SAML and SAML2 issuer signing key. See [2965](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2965).
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,26 @@
...
+- Add validation for SAML and SAML2 algorithm. See [2984](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2984).
```
Comment (msbw2) [inline CHANGELOG.md]:
```suggestion
- Switch to new validation model for SAML and SAML2 algorithm. See [2984](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2984).
```

**Pull request merged**